src/build/arg/mod.rs

```diff
@@ -720,7 +720,7 @@ impl<'help> Arg<'help> {
     /// * supplies the `self` arg.
     /// * supplies *all* of the `names` arguments.
     ///
-    /// **NOTE:** If you wish for this argument to only be required if *one of* these args are
+    /// **NOTE:** If you wish for this argument to only be required unless *one of* these args are
```

Comment:
I think the `if` here is correct.

Or maybe you want to change to `unless *any of* these`.

Reply:
> Or maybe you want to change to `unless *any of* these`.

This feels like the clearest wording to me. Amended in bcd48b570718761cea8b3f12cb0fc91ca644c923